Pinta.Gui.Widgets/Widgets/OpenImages/OpenImagesListWidget.cs

```diff
@@ -142,6 +142,9 @@ void HandleTreeButtonPressEvent (object o, ButtonPressEventArgs args)
 				TreePath path;
 				tree.GetPathAtPos ((int)click_x, (int)click_y, out path);
 
+				if (path == null)
```

Comment:
This can be removed, since I think the bug you were trying to fix was [recently fixed in master](https://github.com/PintaProject/Pinta/commit/1889586e9a1f1a6a3b3e767a9dd58151bc2c6a43)